src/trino/from_unixtime_impl.rs

```diff
@@ -16,27 +16,35 @@
 // under the License.
 
 #![allow(non_camel_case_types)]
-use arrow::datatypes::DataType;
+use arrow::array::{AsArray, TimestampMillisecondArray};
+use arrow::datatypes::{DataType, Int64Type, TimeUnit};
 use datafusion::common::Result;
 use datafusion::error::DataFusionError;
 use datafusion::logical_expr::simplify::{ExprSimplifyResult, SimplifyInfo};
 use datafusion::logical_expr::{ColumnarValue, Expr, ScalarUDFImpl, Signature, Volatility};
 use std::any::Any;
-
-fn from_unixtime_bigint_invoke(_args: &[ColumnarValue]) -> Result<ColumnarValue> {
-    Err(DataFusionError::NotImplemented(format!(
-        "Not implemented {}:{}",
-        file!(),
-        line!()
-    )))
+use std::sync::Arc;
+
+use crate::utils::apply_unary_kernel;
+
+/// Convert `arg[0]`, which should be a BIGINT with the number of seconds since the Unix epoch, 1970-01-01,
+/// to a TIMESTAMP(3), which is the number of milliseconds.
+// TODO?: change the input from BIGINT to DOUBLE, which it is in Trino.
+// TODO?: make sure there is a timezone in the output, which it is in Trino.
```

Comment:
These TODOs should be easy to put in, but are likely to require adjustments at use sites, so the question is whether we want them in now or later?